docs/README.md

```diff
@@ -60,13 +58,27 @@ Kevin Zolman
 
 
 # 💭 Abstract
-<!-- TBD -->
+This application is a Dungeons and Dragons Encounter Builder, with the intended user being a Dungeon Master. It allows the user to fill out a form to add encounter details including encounter name, party size, party level, summary, description, treasure & rewards, and monsters. Monsters from the D&D 5e compendium can be filtered and added based on size, name, hit points, and armor class attributes. Users can also see previews of the encounters they've built and click on the preview to view the full details of the corresponding encounter.
+
 
 # 📝 Context
-This application was built as the group capstone project of Turing School of Software and Design's Front End Web Development program, Mod 4. We were given 2 weeks, from kick-off, to complete and submit the project for evaluation. This repo houses the FE contribution to the project; the BE repo can be found here: [Tavern Keeper BE Repo][backend-repo-url]. The deployed API can be found here: [Tavern Keeper BE Deployment][backend-deployment-url].
+This application was built as the group capstone project of Turing School of Software and Design's Front End Web Development program, Mod 4. We were given 2 weeks from kick-off to complete and submit the project for evaluation. This repo houses the FE contribution to the project; the BE repo and installation instructions can be found here: [Tavern Keeper BE Repo][backend-repo-url]. The deployed API can be found here: [Tavern Keeper BE Deployment][backend-deployment-url].
+
 
 # 🎬 Preview
-<!-- TBD -->
+<details>
+<h2  id="preview"> 🎥 Desktop Preview 📱 </h2>
+<h3>MacBook</h3>
+![macbooktk](https://github.com/Tavern-Keeper-2308/tavern-keeper-fe/assets/139941423/83ab5706-193f-4ee9-a16a-ed9b0c105de5)
+
+<h2  id="preview"> 📱 Mobile and Tablet Views </h2>
+<h3>iPad</h3>
+![ipadtk](https://github.com/Tavern-Keeper-2308/tavern-keeper-fe/assets/139941423/7311f5eb-fdae-408e-b75c-66390a7dae8c)
+
+<h3>iPhone</h3>
+![iphonetk](https://github.com/Tavern-Keeper-2308/tavern-keeper-fe/assets/139941423/5b0aacce-ac4b-4b3e-81bf-0fcc05b0aad6)
+</details>
```

Comment:
None of these previews are actually rendering for me. Are they showing up on the README preview on your machine? All I see are the links themselves.

Reply:
Nope, looking at it again they are being uncooperative. Posting them here in the comment to get the links to try again.
![macbooktk](https://github.com/Tavern-Keeper-2308/tavern-keeper-fe/assets/139941423/385195f5-6e78-4740-bc67-3b9f2bbbf8ec)
![iphonetk](https://github.com/Tavern-Keeper-2308/tavern-keeper-fe/assets/139941423/b39b9af4-d570-4eb6-8daa-00563bf7dc34)
![ipadtk](https://github.com/Tavern-Keeper-2308/tavern-keeper-fe/assets/139941423/a740743c-a714-4edc-bef7-63866c5d7ced)